pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md

```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
-In this case, you want to restore your volume to its latest manual snapshot, taken through the OVHcloud API or your OVHcloud control panel.
+## Instructions
 
-### Requirements
+### Scenario 1: Reverting a volume using a manual snapshot
 
-- You already did a manual snapshot. If not, you can create a `manual` snapshot through the OVHcloud API or your OVHcloud control panel.
-- The manual snapshot has to belong to the volume you want to restore.
+In this case, you want to restore your volume to its latest manual snapshot, created via the OVHcloud API or the OVHcloud Control Panel.
 
-1\. Identify the latest `manual` volume snapshot using the following API route:
->[!api]
+> [!primary]
+> **Requirements for this scenario:**
+>
+> - You already did a manual snapshot. If not, you can create a `manual` snapshot through the OVHcloud API or your OVHcloud control panel.
+> - The manual snapshot has to belong to the volume you want to restore.
+
+1\. Identify the latest `manual` volume snapshot using the following API call:
+
+> [!api]
 >
 > @api {GET} /storage/netapp/{serviceName}/share/{shareId}/snapshot
 >
+
 - `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore 
 
-
 ![RevertManualSnapshot](images/use_case_1_step_1.png){.thumbnail}
 
-2\. Restore your volume to its latest snapshot by calling `/revert` API route 
+2\. Restore your volume to its latest snapshot using the `/revert` API call: 
 
 > [!api]
 >
 > @api {POST} /storage/netapp/{serviceName}/share/{shareId}/revert
 >
-- `{serviceName}` is the service unique ID 
+
+- `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
-- `{snapshotID}` is the latest share snapshot 
+- `{snapshotID}` is the latest share snapshot
 
-The OVHcloud API will return a HTTP 202 (Accepted) and no response body.
+The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>
 The volume status will be set to `reverting` then will go back to `available` once the volume restoration process is complete. At the same time, the snapshot status will be set to `restoring` then will go back to `available` once the volume restoration process is complete.
 
 ![RevertManualSnapshot](images/use_case_1_step_2.png){.thumbnail}
 
+### Scenario 2: Reverting a volume using a snapshot taken via the snapshot policy
 
-## Scenario 2: Revert volume using snapshot taken via snapshot policy
 In this case, a snapshot policy takes regular (automatic) snapshots of a volume. You want to restore your volume to the latest snapshot taken by the policy.
 
-You hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its volume can be restored to it.
-
-### Requirements
+You will have to hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its associated volume can be restored to it.
```

Comment:
> This snapshot becomes `manual`.

This snapshot will become a `manual` snapshot. 

I would change the sentence above to match the `You will have...` from the previous sentence and for more clarity perhaps too.

Reply:
Thanks @dariakts , I've adjusted the wording, let me know if that's better.

Reply:
It's all good for me 👍